src/social-web/routes/feed/inspector.tsx

```diff
@@ -112,6 +118,24 @@ export default function FeedInspector( { id, onClose }: FeedInspectorProps ) {
 					{ ( post.content?.rendered || post.excerpt?.rendered ) && (
 						<RenderHTML html={ post.content?.rendered || post.excerpt?.rendered || '' } />
 					) }
+					{ tags && tags.length > 0 && onTagClick && (
+						<div className="activitypub-inspector-tags">
+							{ tags.map( ( tag: any ) => (
+								<Button
+									key={ tag.id }
+									variant="secondary"
+									size="compact"
```

Comment:
```suggestion
									size="small"
```
How'd you feel to make these look more like on Mastodon with some custom styles? I like how they managed to make them unobtrusive and look like part of the post. To that end, would you consider removing the dividing line above `.activitypub-inspector-tag`?